kernel/vars.mli

```diff
@@ -140,4 +140,6 @@ val subst_univs_level_context : Univ.universe_level_subst -> Constr.rel_context
 val subst_instance_constr : Instance.t -> constr -> constr
 val subst_instance_context : Instance.t -> Constr.rel_context -> Constr.rel_context
 
+val univ_unabstract_constr : Instance.t -> constr univ_abstracted -> constr
```

Comment:
unabstract -> apply ?

Reply:
Only if you insist.

Reply:
`instantiate` would fit even better with the rest of the naming scheme if I may.

Reply:
Indeed!